pkg/google/gke/pricing_map.go

```diff
@@ -272,6 +277,15 @@ func (pm *PricingMap) Populate(ctx context.Context, billingService *billingv1.Cl
 					log.Printf("Storage class %s already exists in region %s", storageClass, data.Region)
 					continue
 				}
+				// Switch statement must go here to handle hyperdisk cases, otherwise what's happening is
+				// The four dimenions get ignored. There is a sku for:
```

Comment:
```suggestion
				// The four dimensions get ignored. There is a sku for:
```

Reply:
lol, I learned today that `golangci-lint` fails on spelling errors, thank you!